dali/test/python/test_video_reader_resize.py

```diff
@@ -0,0 +1,145 @@
+import os
+from functools import partial
+
+import numpy as np
+import nvidia.dali as dali
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+from nvidia.dali.pipeline import Pipeline
+
+
+video_directory = '/tmp/labelled_videos/'
+video_directory_multiple_resolutions = '/tmp/video_resolution/'
+
+pipeline_params = {
+    'num_threads': 8,
+    'device_id': 0,
+    'seed': 0
+}
+
+
+video_reader_params = [{
+    'device': 'gpu',
+    'file_root': video_directory,
+    'sequence_length': 32,
+    'random_shuffle': False
+}, {
+    'device': 'gpu',
+    'file_root': video_directory_multiple_resolutions,
+    'sequence_length': 32,
+    'random_shuffle': False
+}]
+
+
+resize_params = [{
+    'resize_x': 300,
+    'resize_y': 200,
+    'interp_type': types.DALIInterpType.INTERP_CUBIC,
+    'minibatch_size': 8
+}, {
+    'resize_x': 300,
+    'interp_type': types.DALIInterpType.INTERP_CUBIC,
+    'minibatch_size': 8
+}, {
+    'resize_x': 300,
+    'resize_y': 200,
+    'interp_type': types.DALIInterpType.INTERP_LANCZOS3,
+    'minibatch_size': 8
+}, {
+    'resize_shorter': 300,
+    'interp_type': types.DALIInterpType.INTERP_CUBIC,
+    'minibatch_size': 8
+}, {
+    'resize_longer': 500,
+    'interp_type': types.DALIInterpType.INTERP_CUBIC,
+    'minibatch_size': 8
+}, {
+    'resize_x': 300,
+    'resize_y': 200,
+    'min_filter': types.DALIInterpType.INTERP_CUBIC,
+    'mag_filter': types.DALIInterpType.INTERP_TRIANGULAR,
+    'minibatch_size': 8
+}, {
+    'resize_x': 300,
+    'resize_y': 200,
+    'interp_type': types.DALIInterpType.INTERP_CUBIC,
+    'minibatch_size': 4
+}]
+
+
+def video_reader_resize_pipeline(batch_size, video_reader_params, resize_params):
+    pipeline = dali.pipeline.Pipeline(
+        batch_size=batch_size, **pipeline_params)
+
+    with pipeline:
+        outputs = dali.fn.video_reader_resize(
+            **video_reader_params, **resize_params)
+        if type(outputs) == list:
+            outputs = outputs[0]
+        pipeline.set_outputs(outputs)
```

Comment:
Maybe you can use ElementExtract as well here and then just run compare_pipelines.

Reply:
This was suppose to handle the fact, that sometimes you return some meta data for the videos, like labels.

Reply:
Sure